backend-api/src/main/java/org/wcci/apimastery/Controllers/AlbumController.java

```diff
@@ -0,0 +1,23 @@
+package org.wcci.apimastery.Controllers;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.wcci.apimastery.Storage.AlbumRepository;
+import org.wcci.apimastery.models.Album;
+
+import java.util.Collection;
+
+@RestController
+public class AlbumController {
```

Comment:
```suggestion
@RestController
@RequestMapping("/albums")
public class AlbumController {
```
Maybe abstract away any future `@RequestMapping`s to an annotation above the class declaration in the future, less typing when we go to make more endpoints